ui/src/pages/author-guide.tsx

```diff
@@ -80,8 +80,8 @@ const AuthorGuide: NextPage = (): React.ReactElement => (
                         <StandardText>
                             <ul className="ml-8 list-disc">
                                 <li className="mb-8">
-                                    <span className="font-bold">Sign in</span> using your ORCiD credentials (or create
-                                    an ORCiD if you don’t have one). You will also need to provide a verified email
+                                    <span className="font-bold">Sign in</span> using your ORCID® credentials (or create
+                                    an ORCID iD if you don’t have one). You will also need to provide a verified email
```

Comment:
wonder if this should say 

'create an ORCID account if you don't have one'

it was bad before, this doesn't really improve it..

Reply:
Should I change it to "... create an ORCID account if you don't have one ..." ?

Reply:
yes